### PR TITLE
feat: [SRM-13540]: updating image for no data

### PR DESCRIPTION
**Pull request by harshilbits**
### Summary
1. Updating image for no data state of records.
2. Updating image when query is not executed for records.

#### Screenshots
New Note

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/85-cv/pages/health-source/HealthSourceDrawer/component/customiseHealthSource/CustomiseHealthSource.utils.tsx [hunk 1]
```diff
@@ -56,7 +56,7 @@ export const LoadSourceByType = ({
   expressions?: string[]
 }): JSX.Element | null => {
   const isSplunkMetricEnabled = useFeatureFlag(FeatureFlag.CVNG_SPLUNK_METRICS)
-  const isSumoLogicEnabled = useFeatureFlag(FeatureFlag.SRM_SUMO)
+  const isSumoLogicEnabled = useFeatureFlag(FeatureFlag.SRM_SUMO) || true
```
Comment (deepesh-ui) [inline src/modules/85-cv/pages/health-source/HealthSourceDrawer/component/customiseHealthSource/CustomiseHealthSource.utils.tsx:59]: remove this

File: src/modules/85-cv/pages/health-source/HealthSourceDrawer/component/defineHealthSource/DefineHealthSource.tsx [hunk 2]
```diff
@@ -76,7 +76,7 @@ function DefineHealthSource(props: DefineHealthSourceProps): JSX.Element {
   const { isEdit } = sourceData
   const isSplunkMetricEnabled = useFeatureFlag(FeatureFlag.CVNG_SPLUNK_METRICS)
   const isErrorTrackingEnabled = useFeatureFlag(FeatureFlag.CVNG_ENABLED)
-  const isSumoLogicEnabled = useFeatureFlag(FeatureFlag.SRM_SUMO)
+  const isSumoLogicEnabled = useFeatureFlag(FeatureFlag.SRM_SUMO) || true
```
Comment (deepesh-ui) [inline src/modules/85-cv/pages/health-source/HealthSourceDrawer/component/defineHealthSource/DefineHealthSource.tsx:79]: this too

Comment (harshilbits) [reply src/modules/85-cv/pages/health-source/HealthSourceDrawer/component/defineHealthSource/DefineHealthSource.tsx]: done.

File: src/modules/85-cv/pages/health-source/HealthSourceDrawer/component/customiseHealthSource/CustomiseHealthSource.utils.tsx (hunk 1 above)
Comment (harshilbits) [reply src/modules/85-cv/pages/health-source/HealthSourceDrawer/component/customiseHealthSource/CustomiseHealthSource.utils.tsx]: done.

**Review by deepesh-ui — APPROVED** [review]

**Pull request merged**